server/utils/utils.ts

```diff
@@ -44,6 +44,14 @@ export const initialiseName = (fullName?: string): string | null => {
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
 
+export const fullName = (user?: { firstName: string; lastName: string; middleNames?: string }): string | null => {
```

Comment:
Just realised, in the context this is used (the staff member who created the appointment) we should not be displaying the middle name. The first name is shortened to their first initial so as not to give away names to prisoners.

This is a good function and one we could use in other situations. Perhaps rename it fullNameIncludingMiddleNames and have a separate fullName method that doesn't include the middle name